spec/services/scrapers/low_priority_spec.rb

```diff
@@ -4,15 +4,17 @@
 
 RSpec.describe Scrapers::LowPriority do
   describe '#perform' do
-    it 'add specs' do
+    it 'adds jobs to the CourtCaseWorker' do
       test_case = create(:court_case, :with_html, closed_on: 7.months.ago)
       test_case.case_html.update(scraped_at: 7.months.ago)
-
-      data = described_class.perform
+      data = nil
 
       expect do
-        described_class.perform
+        data = described_class.perform
```

Comment:
why are you doing this stuff with the data variable?

Reply:
in order for the second test` expect(data.first.id).to eq test_case.id` to  not error out, its making sure the test_case id equals the id of the first case in the job.